<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_f</class>
 <widget class="QWidget" name="Form_f">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pb_Fimg">
       <property name="text">
        <string>显示频谱图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_filter">
       <property name="text">
        <string>滤波</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>截止频率</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_d0"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>阶数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_n"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>高通-hp,低通-lp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="type">
       <item>
        <property name="text">
         <string>lp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hp</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>滤波器</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="kind">
       <item>
        <property name="text">
         <string>butterworth</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ideal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gaussian</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
